docs/DeveloperGuide.adoc

```diff
@@ -295,60 +295,263 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |new user |See command instruction and usage |Refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new record | Track my spending/income
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |Forgetful user | Add additional details through a description for each record | Better keep track of the details of an expense/income
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |Visually inclined user |Attach an image to each record |Conveniently record additional details of the expense instead of typing it all out
```

Comment:
Is this really a must-have? Can the software still maintain its function without this feature?